### ISSUE TITLE
Incorrect behavior in VMWare Guest

### ISSUE THREAD

**Issue by tschulte** [openstenoproject/plover#707]
Classification: Bug

Reproducibility: Always

## Plover running on Windows Host uses wrong keyboard layout on Linux Guest

Relates to #656. I am running a VMWare Linuxmint VM on my Windows 10 Host. Both OS are configured to use the same keyboard layout (NEO2 in my case -- but the issue is the same with QWERTZ). Plover is running on the Windows Host with an ErgoDox connected using TXBolt protocol over COM3. On the Windows host everything is OK. Independent on the used keyboard layout the output is correct. But on the Linux VM I have to switch to QWERTY for plover to work correctly. If Linux is configured to QWERTZ instead, SO*EPB results in "yone" instead of "zone" (QWERTZ only differs in Z and Y for A-Z). 

## Steps to Reproduce

1. Start Plover on the Host with Steno machine connected using Serial protocol
2. Configure both Host and Guest to use QWERTZ (or AZERTY)
3. Switch to the VM and type something

## Expected Results

1. Plover uses the keyboard layout of the Host instead of QWERTY

## Actual Results

1. Plover seems to expect the VM to use QWERTY

## Version

Plover version 3.1.0.

Installer downloaded from Github.

## Notes

Workaround: Use QWERTY layout in the VM
 
## Configuration

OS: 
Host: Windows 10 Enterprise

Guest: Linux linuxmint 4.4.0-64-generic

**Comment by morinted** [comment]
I don't know if anything Plover can do could fix this behavior.

Just to confirm:

- When Plover isn't working (yone), the regular keyboard does?
- You might try enabling the language bar with keyboard layout indicator to ensure that the layout isn't changing per-window.

**Comment by tschulte** [comment]
Yes, the regular keyboard does work perfectly. I will check tomorrow, if the keyboard layout indicator changes when the VM gets the focus.

**Comment by tschulte** [comment]
When the VM is active, at least the keyboard indicator on Windows does not change. Currently I don't have anything installed on the Windows Host. After I have installed python, I can try to debug what's happening.

**Comment by mkrnr** [comment]
Closing this ticket due to inactivity. Please comment if this is still an issue.
